core/meta/inc/TVirtualStreamerInfo.h

```diff
@@ -25,6 +25,13 @@
 
 #include "ESTLType.h"
 
+#ifdef WIN32
```

Comment:
```suggestion
#ifdef _MSC_VER
```
I prefer `_MSC_VER` to disable it with Visual Studio only